### ISSUE TITLE
Switch recipe is not working

### ISSUE THREAD

**Issue by yaromins** [OpenAgInitiative/openag_ui#108]
Currently, through UI it is possible to select another while another recipe is running.
However, openag_brains reject start_recipe API if another one running:

> {
>   "message": "There is already a recipe running. Please stop it before attempting to start a new one", 
>   "success": false
> }

So I have to manually run `curl --data "" http://localhost:5000/api/0.0.1/service/environment_1/stop_recipe` to stop recipe first and then select a new one.

Shall UI do it automatically?

**Comment by gordonbrander** [comment]
Yes, you're right. We should do this automatically. I think the right way to do this is to have a new backend endpoint that will shut down any currently running recipe.

**Comment by yaromins** [comment]
openag_brains already has endpoint for stop_recipe.

I'm trying to add add this `stop_recipe` API call before `start_recipe` into openag UI.  But so far can't get it working. I've managed to add the stop/start call sequence, but looks like openag_brain needs some delay between `recipe_stop` and `recipe_start`, otherwise `recipe_start` still returns an error. However, my js skills are too limited to understand how to introduce that delay.

Here is the [WIP code I've added so far](https://github.com/OpenAgInitiative/openag_ui/compare/master...serein7:stop_recipe?expand=1)
